### PR TITLE
Menu page after Login

### PR DESCRIPTION
**Pull request by jiwkovva191**
After the user clicks on the Login button, he is being redirected to the Menu page

### CODE REVIEW

File: src/pages-apis/base/Menu.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                
+            
+            
+        </div>
+        </>
+    )
+
+};
+
+export default Menu;
```
Comment (s0ward) [inline src/pages-apis/base/Menu.jsx:36]: add empty line at end of file

File: src/pages-apis/base/Menu.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            <Link to="#" className="menu-page-link">Information</Link>
+           
+            <button className="menu-page-logout" onClick={logout}>Logout</button>
+                
+            
+            
+        </div>
+        </>
+    )
+
```
Comment (s0ward) [inline src/pages-apis/base/Menu.jsx:33]: remove empty line

File: src/style-AK/Menu.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    background-color: #4774e6;
+    border: 1px solid #4774e6;
+    border-radius: 5px;
+    display: block;
+    width: calc(100% - 40px);
+    box-sizing: border-box;
+    transition: 0.3s;
+    cursor: pointer;
+    width: 40vw;
+}
```
Comment (s0ward) [inline src/style-AK/Menu.css:50]: add empty line at end of file

**Comment by jiwkovva191** [comment]
the review is checked and the necessary changes are made

**Review by s0ward — APPROVED** [review]

**Pull request merged**